deployment/ansible/droplet_provision_centos.yml

```diff
@@ -86,14 +75,23 @@
 
   - name: Download and install Docker Compose
     get_url:
-      url: https://github.com/docker/compose/releases/download/1.21.2/docker-compose-Linux-x86_64
+      url: https://github.com/docker/compose/releases/download/1.24.0/docker-compose-Linux-x86_64
       dest: /usr/bin/docker-compose
       mode: 0755
     when:
       - docker_compose_check.msg is defined
       - docker_compose_check.msg.find('No such file or directory') != -1
 
-  - command: "docker-compose -f /root/docker/docker-compose-created.yml up -d"
+  - name: Mount datavolume
+    mount:
+      path: /mnt/datavolumedocker
+      src: /dev/disk/by-id/scsi-0DO_Volume_datavolumedocker
+      opts: discard,defaults,noatime
+      fstype: ext4
+      state: mounted
+    ignore_errors: no
+
+  - command: "docker-compose -f /home/terraformuser/docker/docker-compose-created.yml up -d"
     register: compose_up
 
```

Comment:
- [ ] Restart SSH service, to block future terraformuser logins

Reply:
Done